core/state/plain_state_writer.go

```diff
@@ -145,3 +147,7 @@ func (w *PlainStateWriter) WriteHistory() error {
 func (w *PlainStateWriter) ChangeSetWriter() *ChangeSetWriter {
 	return w.csw
 }
+
+func (w *PlainStateWriter) IncrementVersion() (uint64, error) {
```

Comment:
As with the reader function, this does not need to be wrapped into `PlainStateWriter`, it can be moved to `core/rawdb` package